src/SignalR/clients/ts/FunctionalTests/SignalR.Client.FunctionalTestApp.csproj

```diff
@@ -37,6 +37,11 @@
     <Reference Include="System.Reactive.Linq" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\signalr\signalr.npmproj" />
+    <ProjectReference Include="..\signalr-protocol-msgpack\signalr-protocol-msgpack.npmproj" />
+  </ItemGroup>
```

Comment:
> .dotnet\sdk\3.0.100-preview7-012821\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error NETSDK1004: Assets file 'F:\workspace\_work\1\s\artifacts\obj\SignalR.Client.FunctionalTestApp\project.assets.json' not found. Run a NuGet package restore to generate this file.
Build failed.
Cmd.exe exited with code '1'.

Do you have any idea why nuget restore hasn't run at that point? Or why project.assets.json was deleted?